File: question-one/employees-schedule.kt
fun main(){ 
    busyPeriod(arrayOf("Joy","Ruby"),arrayOf("kakamoga", "kitoko"),10)
    labour(12,5000,9)
}
fun busyPeriod ( employees: Array<String>,  locations: Array<String>, capacity: Int,){
    for (location in locations){
        for ( employee in employees){
            employee == location
        }
    }
    if (employees.size < capacity){
        println("we need more employees")

    }
    else{
        println("employees are enough")
    }
}
fun labour(hoursWorked: Int, salary: Int, lawLabourHours: Int ){
    if (hoursWorked > lawLabourHours){
        var exceedingHours = hoursWorked - lawLabourHours
        var newSalary = (exceedingHours*salary)/100 + salary
        println(newSalary)
    }
    else{
        println(salary)
    }
}

File: question-one/employees-schedule.py
def busy_periods(employees, locations, capacity):
  
    for location in locations:
        count = 0
        for employee in employees:
            if employee[1] == location:
                count += 1
        if count < capacity:
            print(f"We need more employees at {location}")
        else:
            print(f"Employees are enough at {location}")

def labour(hoursWorked, salary, lawLabourHours):
    if hoursWorked > lawLabourHours:
        exceedingHours = hoursWorked - lawLabourHours
        newSalary = (exceedingHours * salary/100) + salary
        print(f"Your new salary is {newSalary}")
    else:
        print(f"Your salary is {salary}")



busy_periods([("Ruby", "Karen"), ("Yordanos", "Karen"), ("Joy", "Korongo"), ("Dana", "Korongo"), ("Eleni", "Korongo")], ["Karen", "Korongo", "Kisumu"], 2)
labour(50, 1000, 40)
